src/containers/LandingPage.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import LoginForm from '../components/LoginForm.jsx';
+import SignupForm from '../components/SignupForm.jsx';
+
+const LandingPage = () => (
+  <div>
+    <div>
+      LoginForm
+      <LoginForm />
+    </div>
+    <br />
```

Comment:
lets do some research on styled components for spacing instead of a new line tag.